web-ui/components/EditorPanelComponents/Console.vue

```diff
@@ -1,11 +1,40 @@
 <template>
-  <div class="output" id="console" v-html="console" />
+  <div class="output" id="console" v-html="computedConsole" />
 </template>
 
 <script>
 export default {
   props: {
-    console: { type: String, default: "" },
+    console: {
+      type: Array,
+      default() {
+        return [];
+      },
+    },
+  },
+  computed: {
+    computedConsole() {
+      let computedConsole = "";
+      this.console.forEach(line => {
+        computedConsole += `<div class = "console-line ${line.type}">${line.output}</div>`;
+      });
+      return computedConsole;
+    },
   },
 };
 </script>
+
+<style>
+.console-line {
+  padding: 5px;
+}
+.console-line:nth-child(2n) {
+  background-color: #e6dfd7;
```

Comment:
Not sure about the alternative line coloring, maybe we should tone the contrast down on it.